starport/pkg/field/field.go

```diff
@@ -3,179 +3,138 @@ package field
 
 import (
 	"fmt"
-	"strings"
 
+	"github.com/tendermint/starport/starport/pkg/datatype"
```

Comment:
Seems like pkg/field also becoming to SDK templates spesific since now it uses the new datatype package.

What do you think merging datatype and field into a single package and placing under templates/?

Reply:
https://github.com/tendermint/starport/pull/1579/commits/06a73332a9ebbf542f4356d9999e4002427d66cb

Nice. I moved the datatype package under the field package and the field under the template package. Now We only need to change the datatype package to add new types. What do you think?